server/protocols/webadmin/cli/james_cli

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python3
+
+import click
+import json
+import requests as req
+import sys
+
+
+def print_json(obj):
+    parsed = json.loads(obj.text)
+    print(json.dumps(parsed, indent=4, sort_keys=True))
+
+
+@click.group()
+@click.option("--ip", default="127.0.0.1", show_default=True, help="Set ip address")
+@click.option("--port", default="8000", show_default=True, help="Set port")
+@click.pass_context
+def main(ctx, ip, port):
+    """Run './james_cli COMMAND --help' for more information on a command."""
+    path = "http://" + ip + ":" + port
+    ctx.obj = {
+        "path": path
+    }
+
+
+# Mapping routes implemantation ---------------------------------------
```

Comment:
very good sign of a lack of structure

Reply:
Wouldn't it be better to separate each component into its own subfile? One for mapping actions, one for domain actions, etc

Reply:
Yes. it's true but I want to focus on making those components runnable together first. Then I will care are about modularity latter.